core/SellCompany.java

```diff
@@ -0,0 +1,87 @@
+package edu.asu.c3simulator.widgets;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton.TextButtonStyle;
+import com.badlogic.gdx.scenes.scene2d.ui.Window;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+/**
+ * Creates a button that in turn creates a popup window when clicked. The popup
+ * window requires confirmation via button click before selling the company.
+ * 
+ * @author Justin
+ *
+ */
+public class SellCompany extends Table
```

Comment:
This isn't a very descriptive class name. Is is a panel? A button? A complex widget?